.github/workflows/create_release.yml

```
@@ -56,6 +56,6 @@ jobs:
           base: dev
           title: Sync to www-covidcast release ${{ steps.version.outputs.next_tag }}
           labels: chore
-          reviewers: krivard
+          assignees: melange396
```

did you wanna keep the commented/empty `reviewers` field?
```suggestion
          # reviewers:
          assignees: melange396
```